src/Keboola/DbExtractor/Configuration/Config.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\Configuration;
+
+use Keboola\Component\Config\BaseConfig;
+
+class Config extends BaseConfig
```

Comment:
#protip - prefixoval bych to na `DbExtractorCommonBaseConfig` (nebo tak něco) - aby se to dalo dobře extendnout. Samotné extractory budou mít asi taky `Config`, tak by se musel aliasovat.

Reply:
A když už tu je, tak by mohl mít nějaké ty metody IMO

Reply:
Nechal bych to tak jak je. Extraktory uz diky pripravovanym zmenam v #94 nebudou muset mit stejny namespace, takze to nijak nebude kolidovat.

Reply:
Nene. Budou. Jde o to, že nejde napsat

```php
// automaticky
use \Full\Namespace\Config;
class Config extends Config {}
```
Ale musíš udělat 
```php 
//ručně
use \Full\Namespace\Config as SomeOtherConfig;
class Config extends SomeOtherConfig {}
```

Reply:
Nadale uz neni duvod aby byly ve stejnem namepsacu. Jakmile se budou upgradovat extraktory, zmenil bych i namespace a na tento `Config` z nich sahat nebudes muset.

Nicmene i tak to prejmenuju, protoze z nich muzeme dedit extractorovy config a tam by to vadilo. Nehlede na to ze neni asi uplne vhodne spolehat na tu zmenu namespacu.